README.md

```diff
@@ -9,3 +9,15 @@ This repository contains postprocessors for SMT-COMP.
 2. Create a tarball with the process wrapper script in the . directory of the tarball.
 
 SMT-COMP 2020 Model validator (experimental)
```

Comment:
Can we update this year?

Reply:
I rewrote the README completely.  No need to mention the year in the README